api/test_token_blacklist.sh

```
@@ -0,0 +1,146 @@
+go build -o bin/blacklist-test-server ./cmd/test-server/main.go
+if [ $? -ne 0 ]; then
+    echo "❌ Build failed"
+    exit 1
+fi
+echo "✅ Build successful"
+echo
+
+# Function to cleanup on exit
+cleanup() {
+    echo "🧹 Stopping test server..."
+    if [ ! -z "$SERVER_PID" ]; then
```

Shellcheck (0.10.0)

[style] Use -n instead of ! -z.

(SC2236)